apps/schematic/api/prepare-python.sh

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
 
-PYTHON_VERSION="3.10.13"
+PYTHON_VERSION="3.10.14"
+
+pyenv update
```

Comment:
@tschaffter Can I add this line to update pyenv to the latest? Otherwise, 3.10.14 can't be downloaded because the older version of pyenv doesn't have 3.10.14 python version available.

Reply:
The version of Python should be pinned to avoid unexpected behavior, unless a strong case can be made to using `latest`. A better solution is to update pyenv in https://github.com/Sage-Bionetworks/sage-monorepo/blob/main/tools/devcontainers/sage/.devcontainer/Dockerfile.

I can update pyenv now as part of the bi-annual tool update.

Reply:
pyenv has been updated

Reply:
Thank you @tschaffter !!